tests/scripts/remote_monitoring_tests/rest_apis/test_update_results.py

```diff
@@ -742,6 +1035,58 @@ def test_update_results_with_valid_and_invalid_interval_duration(test_name, inte
     print("delete exp = ", response.status_code)
 
 
+@pytest.mark.sanity
+@pytest.mark.parametrize("test_name, interval_end_time", interval_end_times)
+def test_update_results_with_valid_and_invalid_interval_duration_for_namespace_experiment(test_name, interval_end_time, cluster_type):
+    """
+    Test Description: This test validates update results by posting results with interval time difference that is not valid for
+    the given measurement duration
+    """
+    input_json_file = "../json_files/create_namespace_exp.json"
+
+    form_kruize_url(cluster_type)
+
+    response = delete_experiment(input_json_file)
+    print("delete exp = ", response.status_code)
+
+    # Create experiment using the specified json
+    response = create_experiment(input_json_file)
+
+    data = response.json()
+    assert response.status_code == SUCCESS_STATUS_CODE
+    assert data['status'] == SUCCESS_STATUS
+    assert data['message'] == CREATE_EXP_SUCCESS_MSG
+
+    # Update results for the experiment
+    result_json_file = "../json_files/update_results_namespace.json"
+
+    json_data = read_json_data_from_file(result_json_file)
+    json_data[0]['interval_end_time'] = interval_end_time
+    json_file = "/tmp/update_results_namespace.json"
+
+    write_json_data_to_file(json_file, json_data)
+
+    response = update_results(json_file)
+
+    data = response.json()
+    if test_name == "valid_plus_30s" or test_name == "valid_minus_30s":
+        assert response.status_code == SUCCESS_STATUS_CODE
+        assert data['status'] == SUCCESS_STATUS
+        assert data['message'] == UPDATE_RESULTS_SUCCESS_MSG
+    elif test_name == "invalid_zero_diff":
+        assert response.status_code == ERROR_STATUS_CODE
+        assert data['status'] == ERROR_STATUS
+        assert data['message'] == 'Out of a total of 1 records, 1 failed to save'
```

Comment:
Define 'Out of a total of 1 records, 1 failed to save' in utils and update all references to use the variable

Reply:
changed this everywhere to a utils variable. 
Also changing     **assert data['message'] == 'Out of a total of 3 records, 3 failed to save'**
the 3 records to a util variable.